cdap-security-spi/pom.xml

```diff
@@ -40,6 +40,11 @@
       <artifactId>cdap-proto</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
```

Comment:
Oh, it's in spi. We should not bring more stuff here as this may collide with plugin code. Let's make a separate class, e.g. `AuthorizedResponse<T>` that has `AuthorizationResult` and `T`